Kube/PrepareNode/README.md

```diff
@@ -0,0 +1,29 @@
+Tasks performed for Ubuntu 18.04:
+
+1. Install Docker dependencies
+1. Install the Docker repository
+1. Install the latest Docker (daemons and CLI)
```

Comment:
Can we specify a version to install?
Installing the latest may be an issue
(if it's not easy, we can do it later)

Reply:
Yes, it is theoretically possible to specify a version, but it is not trivial. Currently we install the latest stable version. By the way, do we really need to install a custom Docker CE on Ubuntu? Is the version of Docker bundled with Ubuntu (currently 18.09.7) really not enough for us? Since we use Kubernetes I think it is the Kubernetes API (and not Docker) what we really depend on (Rancher does depend on Docker, but it works fine with version 18.09.7 according to the documentation).

Reply:
Ok, let's leave it for now and discuss offline if we need to have a specific version or not